src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -43,8 +83,24 @@
   {% set _css_class = _css_class ~ ' ecl-banner--' ~ _size %}
 {% endif %}
 
-{% if _centered %}
-  {% set _css_class = _css_class ~ ' ecl-banner--centered' %}
+{% if _font_size != 'm' %}
+  {% set _css_class = _css_class ~ ' ecl-banner--font-' ~ _font_size %}
+{% endif %}
+
+{% if _font_color != 'dark' %}
```

Comment:
A bit the same as before, we do not check for the value of the parameter to be in the "list" of allowed values, so if we only have two options and we "exclude" one, the other is given, no need to use the string as it comes from the user.